src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test?serverTimezone=UTC";
```

Comment:
The DB_URL is incomplete. It needs to be a full JDBC URL. For example: `jdbc:mysql://localhost:3306/yourdatabasename?serverTimezone=UTC`